### PR TITLE
1768 - Expandable Area IE11 small breakpoint overflow fix

### LINKED ISSUE #1768: Composite Tabs - Show Less overflows in IE11 when browser shrunk
**Describe the bug**
The Show Less button on a composite form overflows in to the tab panel when the browser window is shrunk.

**To Reproduce**
1. Copy the attached [example-composite-form.zip](https://github.com/infor-design/enterprise/files/2946549/example-composite-form.zip) to componets/tabs
2. Navigate to /components/tabs/example-composite-form.html in IE11
3. Shrink the window.

**Expected behavior**
The Show More button should be in the context form, and not overflowed in to the tabs container

**Version**

- ids-enterprise: v4.17.0-dev

**Screenshots**
image

**Platform**
 - OS Version: Windows 10
 - Browser Name: IE 11
 - Browser Version 11.1540.10586.0

### PR DESCRIPTION
**Pull request by davidcarlsonberg**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**
Composite Form was overflowing into lower container in IE11.

**Related github/jira issue (required)**:
Closes #1768.

**Steps necessary to review your pull request (required)**:
- Pull branch
- Run app
- In IE11, browse to http://localhost:4000/components/compositeform/example-index.html
  - Ensure there is no overflow
- In IE11, browse to - Browse to http://localhost:4000/components/tabs/example-composite-form.html
  - Ensure there is no overflow

### CODE REVIEW

**Comment by davidcarlsonberg** [comment]
I did see a `min-height` added inline to `.content` with a comment of `Workaround for IE11` in http://localhost:4000/components/tabs/example-composite-form.html...maybe this is preferred instead of a top level component function?

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -22,6 +22,7 @@
 - `[Accordion]` Fixed a bug where some truncated text elements were not generating a tooltip. ([#1736](https://github.com/infor-design/enterprise/issues/1736))
 - `[Calendar]` Event model title color is not correct if the modal is opened and another event is selected. ([#1739](https://github.com/infor-design/enterprise/issues/1739))
 - `[Calendar]` Modal is still displayed after changing months. ([#1741](https://github.com/infor-design/enterprise/issues/1741))
+- `[Composite Form]` Fix a bug in IE11 where composite form content overflows to lower container. ([#1768](https://github.com/infor-design/enterprise/issues/1768))
```
Comment (tmcconechy) [inline docs/CHANGELOG.md]:
```suggestion
- `[Composite Form]` Fix a bug in IE11 where composite form content overflows to the lower container. ([#1768](https://github.com/infor-design/enterprise/issues/1768))
```

File: src/components/compositeform/compositeform.js [hunk 2]
```diff
@@ -160,6 +164,23 @@ CompositeForm.prototype = {
     return this.element[0].classList.contains('on-side');
   },
 
+  /**
+   * Determines if browser is IE11 and applies min-height fix for overflow
+   * @returns {void}
```
Comment (tmcconechy) [inline src/components/compositeform/compositeform.js]:
```suggestion
   * @private
   * @returns {void}
```

**Comment by davidcarlsonberg** [comment]
Might this make more sense in `expandablearea.js`?

**Comment by tmcconechy** [comment]
It might or maybe even in `animateOpen` if we think its a general issue?

**Pull request merged**